tsnet/tsnet.go

```diff
@@ -113,6 +113,9 @@ type Server struct {
 	// field at zero unless you know what you are doing.
 	Port uint16
 
+	// Additional tags to advertise to Tailnet.
```

Comment:
Please write the docs in the same style as the ones above, starting with the subject being the name of the field.

Also, please match your commit message style to our project's style. We don't use Semantic Commit Messages, nor Slack emoji. Just "tsnet: add Server.AdvertisedTags option" as the title is good.

Thanks!

Reply:
Thanks for the feedback, @bradfitz! Updated the pull request as suggested.

I cannot change the signoff state on the commit as per the instructions. Is there still something I can/need to do to pass that check?

Reply:
`git commit --amend -s` to "sign" the commit for that check to be happy.

Can you squash your three commits down into one with the earlier mentioned commit message title?